src/_4a780.c

```diff
@@ -312,7 +325,21 @@ INCLUDE_ASM("asm/nonmatchings/_4a780", func_8004C674);
 
 INCLUDE_ASM("asm/nonmatchings/_4a780", func_8004C6A0);
 
-INCLUDE_ASM("asm/nonmatchings/_4a780", func_8004C6CC);
+int func_8004C6CC(int* arg1, int* arg2) {
```

Comment:
For future reference, it'd be preferable if temporary variables can be removed when finalizing, and casts (tried) to be put on the right side if needed:

```c
int func_8004C6CC(int* arg1, int** arg2) {
   
    if (arg2[1] != 0) 
    {
        arg2[1][11] = (int)arg2[2];
    }
    return 1;
}
```